test/js/pageobjects/DocumentUpload.js

```diff
@@ -41,58 +43,68 @@ class DocumentUpload extends Base {
   }
 
   async verifyUploaderButton(copy) {
-    const documentUploadScreenStrings = copy.capture
-    verifyElementCopy(this.uploaderBtn, documentUploadScreenStrings.upload_file)
+    const documentUploadStrings = copy.capture
+    verifyElementCopy(this.uploaderBtn, documentUploadStrings.upload_file)
+  }
+  
+  async verifyPassportTitle(copy) {
+    const documentUploadStrings = copy.capture
+    verifyElementCopy(this.title, documentUploadStrings.passport.front.title)
+  }
+
+  async verifyPassportInstructionMessage(copy) {
+    const documentUploadStrings = copy.capture
+    verifyElementCopy(this.uploaderInstructionsMessage, documentUploadStrings.passport.front.instructions)
   }
 
-  async verifyDocumentUploadScreenPassportTitle(copy) {
-    const documentUploadScreenStrings = copy.capture
-    verifyElementCopy(this.title, documentUploadScreenStrings.passport.front.title)
+  async verifyFrontOfDrivingLicenceTitle(copy) {
+    const documentUploadStrings = copy.capture
+    verifyElementCopy(this.title, documentUploadStrings.driving_licence.front.title)
   }
 
-  async verifyDocumentUploadScreenPassportInstructionMessage(copy) {
-    const documentUploadScreenStrings = copy.capture
-    verifyElementCopy(this.uploaderInstructionsMessage, documentUploadScreenStrings.passport.front.instructions)
+  async verifyFrontInstructionMessage(copy) {
```

Comment:
Would be better to make it clear in the name that it is specific to the Driving Licence. Same for the back version of this (i.e. `verifyBackInstructionMessage` on line 75). Also makes the naming convention consistent.

Reply:
Good point. Thanks.